community-tutorials/install-roundcube/01-en.md

```diff
@@ -0,0 +1,287 @@
+
+The tutorial uses the example domain `example.com`. This hostname needs to be replaced by the name of your own domain name when you perform the steps described in this tutorial.
+
+# Requirements
+The installation could also be performed on any VPS but this tutorial focuses on a default webhosting setup:
+
+* Webhosting 1000 or better
+* a (sub) domain
+
+# Step 1 - Adding the subdomain and get the document root
+Adding a subdomain is easy using the WCP [Add Subdomain] button in the webhosting interface "Websites & Domains".
+
+![WCP (plesk) adding a subdomain](images/netcup_wcp_add_subdomain.png)
+
+We are using the following subdomain: wm.example.com
+
+Now we are gathering some information about the webhosting that we need later during the configuration.
+
+Navigate to the WCP "PHP Settings" of the new subdomain and note down the current value of the {DOCROOT} e.g.
+
+`/var/www/vhosts/hosting123456.a1b2c.netcup.net/wm.example.com/httpdocs`
+
+In the "SSL/TLS Certificates" settings of the new subdomain, you can add a recommended SSL certificate with Let's Encrypt.
+
+![WCP (plesk) setup LE SSL certificate](images/netcup_wcp_sslit_lets_encrypt.png)
+
+
+# Step 2 - Download Roundcube and mime.types via SSH
+Downloading the Roundcube release from the Roundcube Download page: [https://roundcube.net/download/]
+
+We are using the stable release 1.5.2 complete package for this tutorial and connect to the webhost using SSH.
+
+Connect via SSH to your webhost using the details in the "Web Hosting Access" section of the subdomain.
+
+Navigate to the document root:
+`cd wm.example.com/httpdocs/`
+
+Delete the favicon.ico and index.html which were created during the add subdomain step:
+`rm favicon.ico index.html`
+
+Download the complete package in tar.gz format:
+`wget https://github.com/roundcube/roundcubemail/releases/download/1.5.2/roundcubemail-1.5.2-complete.tar.gz`
+
+Extract the tar.gz package stripping the subdirectory 'roundcubemail-1.5.2':
+`tar xzvfs roundcubemail-1.5.2-complete.tar.gz --strip-components=1`
+
+Download a mime.types definition from the Apache project:
+`wget http://svn.apache.org/repos/asf/httpd/httpd/trunk/docs/conf/mime.types`
+
+
+# Step 3 - Configuring Roundcube installation with a few fixed settings
+Typically a configuration is created copying from the sample and filling additional setting from the defaults file.
+
+If you feel insecure editing the files with an editor go directly to the next stage or modify the files locally and transfer e.g. via FTP.
+
+Copy from sample configuration:
+`cp config/config.inc.php.sample config/config.inc.php`
+
+Configuring the installation editing the config.inc.php file using your favorite editor. We will be using nano for this instruction.
+`nano config/config.inc.php`
+
+Replacing some configuration options but minimum the following ones:
+- db_dsnw
+- default_host
+- des_key
```

Comment:
maybe tell the user to use `openssl rand -hex 12` to generate that string, to ensure uniqueness of the key.

Reply:
Good idea ... at the moment I am supposed to not touch the PR until the netcup internal review ist done and finishing the setup via web takes care of that. I will keep that idea handy for a follow-up change.